fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/Command.java

```diff
@@ -36,7 +37,7 @@
 /**
  * All DDL and DML commands' super class.
  */
-public abstract class Command extends AbstractPlan implements LogicalPlan {
+public abstract class Command extends AbstractPlan implements LogicalPlan, BlockFD {
```

Comment:
It would be better to make LogicalPlan extends BlockFD

Reply:
No, Implicit implementation is always bad